scripts/doclingTemp.py

```
@@ -0,0 +1,366 @@
+            return seg[k].lower()
+    return ""
+
+
+def pil_crop_from_bbox(page_img: Image.Image, bbox) -> Image.Image:
+    """
+    Crop from a bbox that may be:
+      - dict with x, y, width, height (pixel coords)
+      - tuple/list with (x0, y0, x1, y1) normalized to [0,1]
+    """
+    if isinstance(bbox, dict):
+        x = int(round(bbox.get("x", 0)))
```

Ruff (0.12.2)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)